.gitignore

```diff
@@ -0,0 +1,15 @@
+#  Local .terraform directories
+**/.terraform/*
+
+# .tfstate files
+*.tfstate
+*.tfstate.*
+
+# .tfvars files
+*.tfvars
+
+
+# JetBrains
```

Comment:
after setting up the global .gitignore you can remove the JetBrains entries from this file

Reply:
done on Marco's IDE, file fixed